app/controllers/revisions_controller.rb

```diff
@@ -52,28 +60,29 @@ def load_node
     initialize_nodes_sidebar
   end
 
-  def load_record
-    @record = if params[:card_id]
-                @list.cards.find(params[:card_id])
-              elsif params[:evidence_id]
-                @node.evidence.find(params[:evidence_id])
-              elsif params[:issue_id]
-                Issue.find(params[:issue_id])
-              elsif params[:note_id]
-                @node.notes.find(params[:note_id])
-              else
-                raise 'Unable to identify record type'
-              end
+  def set_variables_from_params
+    if params[:card_id]
+      load_list
+      @path = sidebar_project_board_list_card_revisions_path
```

Comment:
I don't think we need an extra "sidebar" route for every type of item. A single one will suffice  like: `sidebar_project_revisions_path(params)`. That way, we can pass the same `@path` value for each type.